### PR TITLE
Mobile Menu

### PR DESCRIPTION
**Pull request by Franky237**
- Added simple JavaScript to the menu
- Added click option to display nav items
- Added click option to close nav items
- Added click option to redirect to the area on the page

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,6 +1,8 @@
 @import url('https://fonts.googleapis.com/css?family=Poppins&display=swap');
 
 * {
+  padding: 0;
+  margin: 0;
   box-sizing: border-box;
 }
```
Comment (christianbiring1) [inline style.css:7]: - [ ] For a great user experience, kindly make sure you stick to the design as much as possible. (typos, spacing, color)

Your design :point_down: 
image
Figma template :point_down: 
image

File: style.css [hunk 2]
```diff
@@ -9,7 +11,7 @@ body {
   background-color: #e5e5e5;
 }
 
-@media all and (max-width: 1020) {
+@media all and (max-width: 768px) {
   body {
     padding: auto;
```
Comment (christianbiring1) [inline style.css]: - [ ] Kindly stick to the design as much as possible. for a better user experience. 
`hint: You may consider adding the font of your elements and readjust the placement of elements on your page.`

Your design :point_down: 
image

Figma design :point_down: 
image

File: style.css [hunk 3]
```diff
@@ -18,90 +20,131 @@ body {
   }
 }
 
+a {
+  color: black;
+  text-decoration: none;
+}
+
 .navbar {
```
Comment (christianbiring1) [inline style.css:25]: - [ ] Kindly make sure you stick to the design as much as possible.
Your design :point_down: 
image
Figma design :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Franky237 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
almost


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `summary` :+1: 
- [ ] Good commit history.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Giving your PR a descriptive PR `title`
`hint: Mobile menu would sound great than Add JavaScript to menu.`
- Also, make sure your design both desktop and mobile look similar to the ones provided in Figma.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mckent05 — APPROVED** [review]
Hi @Franky237,

# STATUS: APPROVED ✔️ ✔️  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ 
- Good job making changes suggested by previous reviewer 👍 

## Optional suggestions
- Your project (mobile version) has a  scroll on the x-axis, please fix this 🙏 

Your project:
dgn


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**